### ISSUE TITLE
Mods do not install without user intervention

### ISSUE THREAD

**Issue by unixbird** [Hermsi1337/docker-ark-server#12]
When using the docker-compose examples and inserting your mods ie GAME_MOD_IDS="569786012,731604991,899250777,774762563" etc 

It then informs you that you must install said mods with the ark manager via docker exec. Is this intended?

**Comment by Hermsi1337** [comment]
Hey, sorry for my very delayed response.

Actually this is NOT how it is intended. I'll provide a fix for this.

**Comment by unixbird** [comment]
Awesome! It'll make it much easier to put in a proper cluster if that gets fixed. I've also noticed i need to manually run the updates as well sometimes

**Comment by bzyg7b** [comment]
Still seems to be an issue for me even in #25 & also I am finding even aftaer manualy installing the mos only one is loaded and if I exec inside the container I get this:
```
Running command 'list-mods' for instance 'main'
   895711211: [Content/Mods]       Classic Flyers
/usr/local/bin/arkmanager: line 2446: printf: 731604991"): invalid number
   731604991: [ark_GameModIds]     Structures Plus (S+)
          56: [ark_GameModIds]     
   731604991: [Content/Mods]       Structures Plus (S+)
```
Not sure if this is the fault of arkmanager or the continer or my config.
I have the following line in my compose file and it seems to load only the first mod in the list, if I switch them around it still just laods the first in the list.
`- GAME_MOD_IDS="895711211,731604991")`

**Comment by Hermsi1337** [comment]
Fixed by #25

**Comment by bzyg7b** [comment]
> Still seems to be an issue for me even in #25 & also I am finding even aftaer manualy installing the mos only one is loaded and if I exec inside the container I get this:
> 
> ```
> Running command 'list-mods' for instance 'main'
>    895711211: [Content/Mods]       Classic Flyers
> /usr/local/bin/arkmanager: line 2446: printf: 731604991"): invalid number
>    731604991: [ark_GameModIds]     Structures Plus (S+)
>           56: [ark_GameModIds]     
>    731604991: [Content/Mods]       Structures Plus (S+)
> ```
> 
> Not sure if this is the fault of arkmanager or the continer or my config. I have the following line in my compose file and it seems to load only the first mod in the list, if I switch them around it still just laods the first in the list. `- GAME_MOD_IDS="895711211,731604991")`
> 
> Edit: might have found the issue, I included the `)` from the README so might be my bad, I will test this now

Update to the above the issue was both quotes and bracket setting the docker-compose file to the following resolved it:
`- GAME_MOD_IDS=895711211,731604991`
